addons/markers/script_component.hpp

```diff
@@ -24,3 +24,1014 @@
     localize "str_channel_vehicle", \
     localize "str_channel_direct" \
 ]
+
+#define ACE_MAP_CLASS(MAPBASE,MAPID) \
+    class MAPBASE##_##MAPID : MAPBASE { \
+        scope = 1; \
+        scopeArsenal = 0; \
+        scopeCurator = 0; \
+        descriptionShort = QUOTE(MAPID); \
+        GVAR(mapID) = MAPID; \
+    };
+
+#define ACE_MAPS(MAPBASE) \
```

Comment:
These macros are usually inserted into `CfgWeapon.hpp` directly. 

Also the `ACE_MAPS` macro seems redundant to me.